FreeAPS/Sources/Localizations/Main/en.lproj/Localizable.strings

```diff
@@ -2031,7 +2031,7 @@ Enact a temp Basal or a temp target */
 "Insulin Peak Time" = "Insulin Peak Time";
 
 /* "Insulin Peak Time" */
-"Time of maximum blood glucose lowering effect of insulin, in minutes. Beware: Oref assumes for ultra-rapid (Lyumjev) & rapid-acting (Fiasp) curves minimal (35 & 50 min) and maximal (100 & 120 min) applicable insulinPeakTimes. Using a custom insulinPeakTime outside these bounds will result in issues with Trio, longer loop calculations and possible red loops." = "Time of maximum blood glucose lowering effect of insulin, in minutes. Beware: Oref assumes for ultra-rapid (Lyumjev) & rapid-acting (Fiasp) curves minimal (35 & 50 min) and maximal (100 & 120 min) applicable insulinPeakTimes. Using a custom insulinPeakTime outside these bounds will result in issues with Trio, longer loop calculations and possible red loops.";
+"Time of maximum blood glucose lowering effect of insulin, in minutes. Beware: Oref assumes for ultra-rapid (Lyumjev & Fiasp) & rapid-acting (Novolog/Novorapid, Humalog, & Apidra) curves minimal (35 & 50 min) and maximal (100 & 120 min) applicable insulinPeakTimes. Using a custom insulinPeakTime outside these bounds will result in issues with Trio, longer loop calculations and possible red loops." = "Time of maximum blood glucose lowering effect of insulin, in minutes. Beware: Oref assumes for ultra-rapid (Lyumjev & Fiasp) & rapid-acting (Novolog/Novorapid, Humalog, & Apidra) curves minimal (35 & 50 min) and maximal (100 & 120 min) applicable insulinPeakTimes. Using a custom insulinPeakTime outside these bounds will result in issues with Trio, longer loop calculations and possible red loops.";
```

Comment:
I still think this is horribly confusing. There is an Insulin Type selected in Pump; Insulin Curve selected in Preferences and then this. And what the heck is bi-linear (Insulin Curve in preferences)? 

Also the exponential model (for rapid-acting and ultra-rapid) has 3 parameters - this is only changing one.

What about something like this, where you make sure the limits are right and fix the {pump/insulin curve} section:

Custom adjustment for time of maximum blood glucose lowering effect of insulin, in minutes. Most people should use the insulin model as selected in {the pump or preferences: Insulin curve}. If custom adjustment is desired, limit the range to be no shorter than 35 minutes and no longer than 120 minutes.

 Beware: Oref assumes for ultra-rapid (Lyumjev & Fiasp) & rapid-acting (Novolog/Novorapid, Humalog, & Apidra) curves minimal (35 & 50 min) and maximal (100 & 120 min) applicable insulinPeakTimes. Using a custom insulinPeakTime outside these bounds will result in issues with Trio, longer loop calculations and possible red loops." = "Time of maximum blood glucose lowering effect of insulin, in minutes. Beware: Oref assumes for ultra-rapid (Lyumjev & Fiasp) & rapid-acting (Novolog/Novorapid, Humalog, & Apidra) curves minimal (35 & 50 min) and maximal (100 & 120 min) applicable insulinPeakTimes. Using a custom insulinPeakTime outside these bounds will result in issues with Trio, longer loop calculations and possible red loops.

Reply:
To answer this in short: the insulin curve setting can be deleted, it’s useless. Insulin is selected via pump manager UI from LoopKit.

The text reads good!